crt_portal/cts_forms/templates/forms/report_police_location.html

```diff
@@ -0,0 +1,38 @@
+<fieldset class="usa-fieldset margin-bottom-4">
+  <legend class="em-text">
+    Where did this take place?
+  </legend>
+  <div data-toggle="toggle">
+    {{ wizard.form.inside_correctional_facility }}
+  </div>
+  {% with field=wizard.form.correctional_facility_type %}
+    <div class="other-class-option padding-left-4 margin-bottom-2 padding-top-3">
+      <div class="blue-left-highlight">
+        <label for="{{ field.id_for_label }}">
+          <b>{{ field.help_text }}</b>  
+        </label>
+        {{ field }}
+      </div>
+    </div>
+  {% endwith %}
+
```

Comment:
We need the error for the field here. The required error is not showing up under the question.